drivers/system_pathways/app/models/system_pathways/equity/race_and_ethnicity.rb

```diff
@@ -0,0 +1,117 @@
+    @race_and_ethnicity_counts ||= node_names.map do |label|
+      counts = race_ethnicity_combinations.values.map { |m| [m, 0] }.to_h
+      # Get rid of the distinct from node_clients
+      scope = SystemPathways::Enrollment.joins(:client).where(id: node_clients(label).select(:id))
+      race_data = pluck_to_hash(race_columns.except('race_none').map { |k, v| [k, v] }.to_h, scope)
+      race_ethnicity_combinations.each do |key, value|
+        counts[value] = lookups[key]&.call(race_data)
+      end
+      [
+        label,
+        counts,
+      ]
+    end.to_h
+  end
+
+  private def single_race(race_data, race)
+    race_data.select { |client| client[race] && !client['hispanic_latinaeo'] }.
+      map { |client| client.except('id', 'hispanic_latinaeo').values }.
+      count { |m| m.count(true) == 1 }
+  end
+
+  private def single_race_latinaeo(race_data, race)
+    race_data.select { |client| client[race] && client['hispanic_latinaeo'] }.
+      map { |client| client.except('id', 'hispanic_latinaeo').values }.
+      count { |m| m.count(true) == 1 }
+  end
+
+  private def lookups
+    {
+      am_ind_ak_native: ->(race_data) { single_race(race_data, 'am_ind_ak_native') },
+      am_ind_ak_native_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'am_ind_ak_native') },
+      asian: ->(race_data) { single_race(race_data, 'asian') },
+      asian_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'asian') },
+      black_af_american: ->(race_data) { single_race(race_data, 'black_af_american') },
+      black_af_american_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'black_af_american') },
+      hispanic_latinaeo: ->(race_data) do
+        race_data.select { |client| client['hispanic_latinaeo'] }.
+          map { |client| client.except('id').values }.
+          count { |m| m.count(true) == 1 }
+      end,
+      mid_east_n_african: ->(race_data) { single_race(race_data, 'mid_east_n_african') },
+      mid_east_n_african_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'mid_east_n_african') },
+      native_hi_pacific: ->(race_data) { single_race(race_data, 'native_hi_pacific') },
+      native_hi_pacific_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'native_hi_pacific') },
+      white: ->(race_data) { single_race(race_data, 'white') },
+      white_hispanic_latinaeo: ->(race_data) { single_race_latinaeo(race_data, 'white') },
+
+      multi_racial: ->(race_data) do
+        race_data.reject { |client| client['hispanic_latinaeo'] }.
+          map { |client| client.except('id', 'hispanic_latinaeo').values }.
+          count { |m| m.count(true) > 1 }
+      end,
+
+      multi_racial_hispanic_latinaeo: ->(race_data) do
+        race_data.select { |client| client['hispanic_latinaeo'] }.
+          map { |client| client.except('id', 'hispanic_latinaeo').values }.
+          count { |m| m.count(true) > 1 }
+      end,
```

Comment:
These could probably use a comment as well, or maybe just move the `hispanic_latinaeo:` down here, and add one comment for all 3.